android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -189,6 +190,7 @@ public void cropImage(
         (int) offset.getDouble("y"),
         (int) size.getDouble("width"),
         (int) size.getDouble("height"),
+        useExternalCache,
```

Comment:
given we're trying to avoid using external cache, `useInternalCache` seems like a more logical flag to add. That said, this should work

Reply:
I agree, let's change it

Reply:
@mvayngrib  Fixed